conformance/resources/manifests/manifests.yaml

```diff
@@ -66,7 +54,95 @@ spec:
   - name: http
     port: 80
     protocol: HTTP
-    hostname: "secondary.example.com" # Distinct hostname to differentiate from conformance-gateway
+    hostname: "secondary.example.com"
```

Comment:
I don't see a hostname configured on the primary Gateway. Why is a hostname only required on the secondary Gateway?

Reply:
Having no hostname, acts as a wildcard matcher for all incoming traffic, all traffic other than `secondary.example.com` will be routed through primary gateway.